src/Controller/Api/CategoryController.php

```diff
@@ -275,7 +275,7 @@ protected function getCategory($id)
         $category = $this->categoryManager->find($id);
 
         if (null === $category) {
-            throw new NotFoundHttpException(sprintf('Category (%s) not found', $id));
+            throw new NotFoundHttpException(sprintf('Category not found for identifier %s.', var_export($id, true)));
```

Comment:
Shall we use a named ctor?

```php
NotFoundHttpException implements NotFoundHttpException
{
    public static function forIdentifier(string $identifier)
....
}
```

??

Reply:
This Expection should be provide in 7 bundles (https://github.com/sonata-project/dev-kit/issues/778). IMO it is another reason to create `sonata-project/contracts`. 

For now I will keep current version.